test/parallel/test-stream-readable-error-end.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const common = require('../common');
+const { Readable } = require('stream');
+const assert = require('assert');
+
+{
+  // Ensure 'end' is not emitted after 'error'.
+  // This test is slightly more complicated than
+  // needed in order to better illustrate the invariant.
+
+  const r = new Readable({ read() {} });
+
+  let errorEmitted = false;
+
+  r.on('data', common.mustCall());
+  r.on('end', () => {
```

Comment:
`common.mustCall()`?

Reply:
It shouldn’t be called. I still add this here since this test just makes sure that if it is called, it’s called before error

Reply:
I don't think it is very useful.  If this test has a deterministic behavior we should test it with `common.must{,Not}Call()`.

Reply:
Ok. Fixed.